encodings/fastlanes/src/bitpacking/serde.rs

```diff
@@ -5,19 +5,21 @@ use vortex_array::variants::PrimitiveArrayTrait;
 use vortex_array::vtable::EncodingVTable;
 use vortex_array::{
     Array, ArrayBufferVisitor, ArrayChildVisitor, ArrayContext, ArrayExt, ArrayRef,
-    ArrayVisitorImpl, Canonical, DeserializeMetadata, Encoding, EncodingId, RkyvMetadata,
+    ArrayVisitorImpl, Canonical, DeserializeMetadata, Encoding, EncodingId, ProstMetadata,
 };
 use vortex_dtype::{DType, PType};
 use vortex_error::{VortexError, VortexExpect, VortexResult, vortex_bail, vortex_err};
 
 use super::{BitPackedEncoding, bit_width_histogram, find_best_bit_width};
 use crate::{BitPackedArray, bitpack_encode};
 
-#[derive(Debug, Clone, rkyv::Archive, rkyv::Serialize, rkyv::Deserialize)]
-#[repr(C)]
+#[derive(Clone, prost::Message)]
 pub struct BitPackedMetadata {
-    pub(crate) bit_width: u8,
-    pub(crate) offset: u16, // must be <1024
+    #[prost(uint32, tag = "1")]
+    pub(crate) bit_width: u32,
+    #[prost(uint32, tag = "2")]
+    pub(crate) offset: u32, // must be <1024
+    #[prost(message, tag = "3")]
```

Comment:
`optional`? (that's what we did for e.g., patches in ALPMetadata, should make sure we're consistent)

Reply:
seems like optional is unnecessary since the value is not a primitive type but can add it for completness since that's what code generator produces